GETTING_STARTED.md

```diff
@@ -0,0 +1,602 @@
+# Getting started with kitchen-terraform
+
+Hello!  This tutorial will walk you through setting up a terraform config to spin up an AWS EC2 instance using Inspec and kitchen-terraform.
+
+Our Tools
+* [Terraform](https://www.terraform.io/)
+* [Test Kitchen](https://github.com/test-kitchen/test-kitchen)
+* [InSpec](https://github.com/chef/inspec)
+* [kitchen-terraform](https://github.com/newcontext/kitchen-terraform)
+
+Make sure you have the following prerequisites for this tutorial
+
+Prerequisites
+* An AWS Account
+* An AWS Access Key ID
+* An AWS Secret Key
+* An AWS Keypair - you will need the private key of this pair somewhere on your workstation
+* Terraform installed
+* Bundler installed
+* Ruby 2.3.1 installed and used (use your favorite ruby version manager, I used [rvm](https://rvm.io/) when creating this tutorial)
+
+So let's start building this config from scratch!
+
+## Setting up our development environment
+
+First, let's create a new directory for our config
+
+```
+  $ mkdir tf_aws_cluster
+```
+
+And cd into that directory
+
+```
+  $ cd tf_aws_cluster
+```
+
+Now, create some skeleton terraform files
+
+```
+  $ touch main.tf variables.tf output.tf testing.tfvars
+```
+
+Then create our test kitchen scaffolding
```

Comment:
This step doesn't really provide any value from the tutorial perspective and leads to more convoluted instructions when the existing `kitchen.yml` has to be changed rather than just created.